public/js/clients/firefox/events.js

```diff
@@ -62,6 +62,11 @@ function newSource(_, packet) {
   if (NEW_SOURCE_IGNORED_URLS.indexOf(source.url) > -1) {
     return;
   }
+
+  if (source.introductionType == "debugger eval") {
```

Comment:
@jlongster I'm not sure where the regression is, but I'm seeing this:

![screen shot 2016-08-08 at 3 01 14 pm](https://cloud.githubusercontent.com/assets/254562/17492234/e35661f6-5d78-11e6-8f7b-a1835422fe2e.png)